src/clr-angular/utils/drag-and-drop/draggable.spec.ts

```diff
@@ -41,24 +92,168 @@ export default function(): void {
                 expect(this.dragHandleRegistrar.customHandleEl).toBeUndefined();
             });
 
+            it("should instantiate cloned version of draggable as ghost on drag start", function() {
+                this.dragEventListener.dragStarted.next();
+                expect(this.fixture.nativeElement.querySelectorAll("clr-draggable-ghost").length).toBe(1);
+                const draggableGhost = this.fixture.nativeElement.querySelector("clr-draggable-ghost");
```

Comment:
nit: Instead of `querySelectorAll` and then `querySelector` to `const draggableGhost`, why not just `querySelectorAll` to `const draggableGhost` and use the first child of it everywhere after testing that the length of the result of `querySelectorAll` is 1

Reply:
Address this.